src/app/(home)/domains/[id]/page.tsx

```diff
@@ -1,279 +1,262 @@
-                            >
-                                {t('welcome')}
-                            </Button>
-                            <Button
-                                variant="contained"
-                                color="primary"
-                                onClick={() => handleDomainActions(id, "start")}
-                                disabled={
-                                    domain.status === "online" ||
-                                    domain.dbInitialized === false ||
-                                    actionInProgress
-                                }
-                            >
-                                {t('start')}
-                            </Button>
-                            <Button
-                                variant="outlined"
-                                color="error"
-                                onClick={() => handleDomainActions(id, "stop")}
-                                disabled={
-                                    domain.status === "not running" ||
-                                    domain.dbInitialized === false ||
-                                    actionInProgress}
-                            >
-                                {t('stop')}
-                            </Button>
-                            <Button
-                                variant="contained"
-                                color="error"
-                                onClick={() => handleDomainActions(id, "destroy")}
-                                disabled={
-                                    domain.status === "online" ||
-                                    domain.dbInitialized === false ||
-                                    actionInProgress}
-                            >
-                                {t('destroy')}
-                            </Button>
-                            <Button variant="contained" color="primary" href={`/domains/${id}/edit`}>
-                                {t('edit')}
-                            </Button>
-                        </Box>
-                    </Grid>
-                </Grid>
-                <Typography variant="h6">
-                    {t('basicData')}
-                </Typography>
-                <RenderDomainProperties properties={domainProperties} />
-                <Typography variant="h6">
-                    {t('advancedSettings')}
-                </Typography>
-                {Object.keys(variables).length === 0 ? (
-                    <RenderDomainProperties properties={{ "None defined": "" }} />
-                ) : (
-                    <RenderDomainProperties properties={variables} />
-                )}
+		if (domain.environments) {
+			domain.environments
+				.sort((a, b) => a.variable.localeCompare(b.variable))
+				.forEach((env) => {
+					variables = { ...variables, [env.variable]: env.value };
+				});
+		}
+	}
 
-            </Box>
+	const adminActions: ActionsMenuItem[] = [
```

Comment:
These actions are the same as those triggered by the buttons for init, start, stop, greet, and destroy.
They are passed as props to the `NSActionsMenu` component, which renders a responsive menu. On small screens, it appears as a quick actions button in the lower-right corner.